reports/get_all_extensions.py

```diff
@@ -0,0 +1,111 @@
+"""
+Script to collect metadata on all published extensions. Requires an access token 
+from https://open-vsx.org/user-settings/tokens. Used by Jupyter notebooks.
+"""
+import requests
+import os
+import json
+import time
+from datetime import datetime
+
+API_ENDPOINT = "https://open-vsx.org/"
+
+ACCESS_TOKEN = os.getenv('ACCESS_TOKEN')
+JSON_FILENAME = 'extensions.json'
+TSV_FILENAME = 'extensions.tsv'
+
+url = API_ENDPOINT + 'api/-/search?size=100&token=%s' % ACCESS_TOKEN
```

Comment:
The search endpoint is public. You don't need a token.

Reply:
Yep, removing it